README.md

```diff
@@ -1,6 +1,6 @@
 # GithubKitForSample
 
-This is simple Github API client and UI for using in sample projects.
+This is simple Github API client and UI to use in sample projects.
```

Comment:
Ah, but you missed adding "a" between "is" and "simple."

Reply:
Good catch
#2